services/venice-server/src/main/java/com/linkedin/venice/stats/ServerHttpRequestStats.java

```diff
@@ -132,73 +151,215 @@ public ServerHttpRequestStats(
        */
       largeValueLookupStats.add(new Rate());
     }
-    multiChunkLargeValueCountSensor = registerSensor(
+    // multiChunkLargeValueCountSensor = registerSensor(
+    // "storage_engine_large_value_lookup",
+    // largeValueLookupStats.toArray(new MeasurableStat[largeValueLookupStats.size()]));
+    multiChunkLargeValueCountSensor = registerPerStoreAndTotal(
         "storage_engine_large_value_lookup",
+        totalStats,
+        () -> totalStats.multiChunkLargeValueCountSensor,
         largeValueLookupStats.toArray(new MeasurableStat[largeValueLookupStats.size()]));
 
     Rate requestKeyCount = new OccurrenceRate();
     Rate successRequestKeyCount = new OccurrenceRate();
-    requestKeyCountSensor = registerSensor("request_key_count", new Rate(), requestKeyCount, new Avg(), new Max());
-    successRequestKeyCountSensor =
-        registerSensor("success_request_key_count", new Rate(), successRequestKeyCount, new Avg(), new Max());
-    requestSizeInBytesSensor = registerSensor("request_size_in_bytes", new Avg(), new Min(), new Max());
+    requestKeyCountSensor = registerPerStoreAndTotal(
+        "request_key_count",
+        totalStats,
+        () -> totalStats.requestKeyCountSensor,
+        new Rate(),
+        requestKeyCount,
+        new Avg(),
+        new Max());
+    successRequestKeyCountSensor = registerPerStoreAndTotal(
+        "success_request_key_count",
+        totalStats,
+        () -> totalStats.successRequestKeyCountSensor,
+        new Rate(),
+        successRequestKeyCount,
+        new Avg(),
+        new Max());
+    requestSizeInBytesSensor = registerPerStoreAndTotal(
+        "request_size_in_bytes",
+        totalStats,
+        () -> totalStats.requestSizeInBytesSensor,
+        new Avg(),
+        new Min(),
+        new Max());
```

Comment:
Look at `HostLevelIngestionStats::newAvgAndMax`. Maybe this should be pulled up into `AbstractVeniceStats` along with other common combinations like `newAvgMinAndMax()`. These could be made protected, so all sub-classes can access it. This can reduce boilerplate a bit.

Reply:
Actually the code for percentile we have is 

  private Sensor getPercentileStatSensor(String name) {
    return registerSensor(
        name,
        TehutiUtils.getPercentileStat(getName(), getFullMetricName(name)),
        new Avg(),
        new Max());
  }

which is basically 5 metrics (3 percentile + avg + max). so I can pull that into `AbstractVeniceStats` with a new name. But do we need both P50 and avg ? IIRC we do use max sometimes for debugging pathological cases.